content/02.main-text.md

```diff
@@ -2,12 +2,12 @@
 
 The internet enables science to be shared in real-time at a low cost to a global audience.
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
-However, the scientific community requires tools whose workflows encourage openness rather than stifle it.
-For example, manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, where anyone is able to contribute and the contribution history is preserved and public.
-Hence, we report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
+However, the scientific community requires tools whose workflows encourage openness.
+Manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, by which anyone is able to contribute and the contribution history is preserved and public.
+We report a new tool and workflow for authoring scholarly manuscripts in the open, as well as the collaborative project that led to its creation.
 
 Open scholarly writing, a form of crowdsourcing [@pmcid:PMC4719068], has particular benefits for review articles, which present the state of the art in a scientific field [@doi:10.1371/journal.pcbi.1003149].
-Literature reviews are typically written in private by a closed team of colleagues.
+Literature reviews are typically written in private by an invited team of colleagues.
```

Comment:
I'm not sure they are actively `closed` so much as passively closed. Thus why I switched the phrasing.